lib/preflight_check.js

```diff
@@ -4,7 +4,11 @@ const requiredEnvVars = {
   'SCSB_URL': process.env.SCSB_URL,
   'SCSB_API_KEY': process.env.SCSB_API_KEY,
   'ELASTICSEARCH_HOST': process.env.ELASTICSEARCH_HOST,
-  'RESOURCES_INDEX': process.env.RESOURCES_INDEX
+  'RESOURCES_INDEX': process.env.RESOURCES_INDEX,
+  'NYPL_API_BASE_URL': process.env.NYPL_API_BASE_URL,
+  'NYPL_OAUTH_URL': process.env.NYPL_OAUTH_URL,
+  'NYPL_OAUTH_ID': process.env.NYPL_OAUTH_ID,
+  'NYPL_OAUTH_SECRET': process.env.NYPL_OAUTH_SECRET
 }
```

Comment:
## GARF!

Good catch - I added a `console.log()` for now.
I made a decent commit message in 83dfaf92d25ee4fc5c6cb6fb81e0ebd699baae5c.